src/parser.cpp

```diff
@@ -3,33 +3,19 @@
 
 namespace ada {
 
-  Parser::Parser(const std::string_view input, std::optional<ada::URL> optional_base_url, std::optional<ada::encoding_type> encoding_override,
-                 std::optional<ada::state> state_override) {
-    // Assign base_url if it exists
-    base_url = optional_base_url;
+  Parser::Parser(const std::string_view input, std::optional<ada::URL> optional_base_url, 
+                 std::optional<ada::encoding_type> encoding_override,
```

Comment:
Is there a way to enforce either this indentation or the previous one through a Github Workflow?

Reply:
I am dropping this PR.